luigi/configuration/toml_parser.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Cindicator Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os.path
+
+try:
+    import toml
+except ImportError:
+    toml = False
```

Comment:
This will end up following error if you have `toml` library not installed...
Works great after installation!

```
File "/Users/*****/.pyenv/versions/miniconda-latest/envs/ddhr-rjb/lib/python3.6/site-packages/luigi/configuration/toml_parser.py", line 53, in read
    self.data = self._update_data(self.data, toml.load(path))
AttributeError: 'bool' object has no attribute 'load'
```

Reply:
Oh noes, @orsinium can you fix this asap?